fdi/findings_data_import.py

```diff
@@ -133,11 +134,6 @@ def import_data(
     temp_file_descriptor, temp_data_filepath = tempfile.mkstemp()
 
     try:
```

Comment:
A potential improvement is to wrap each piece of code that could throw an exception in its own `try` block, and catch only the exceptions that can be thrown in the corresponding `catch` block.  That way you can give a context-specific error or warning message when you catch the exception, and be more specific about which exceptions you are catching.

This doesn't need to hold up this PR, but it's an idea for a future improvement.

Reply:
@mzack5020 or @mcdonnnj - Can one of you please add a brief issue to this repo with @jsf9k's suggested improvement?

Reply:
Please see #15 for future work.